integration-tests/suites/image_json.go

```diff
@@ -28,6 +28,5 @@ func (s *ImageLabelJSONTestSuite) TestRunImageWithJSONLabel() {
 }
 
 func (s *ImageLabelJSONTestSuite) TearDownSuite() {
-	s.StopCollector()
```

Comment:
I think this is still required, because we're starting collector on line 13 (or we can remove line 13 if collector is not needed for this test)